beacon_node/execution_layer/src/lib.rs

```diff
@@ -157,6 +157,17 @@ impl From<ApiError> for Error {
     }
 }
 
+impl From<EngineError> for Error {
+    fn from(e: EngineError) -> Self {
+        match e {
+            // This removes an unnecessary layer of indirection.
+            // TODO (mark): consider refactoring these error enums
```

Comment:
Just noting this in case it was forgotten

Reply:
I was noting this for a future PR but thanks for reminding me so I can put it on my list :)